content/en/security/sensitive_data_scanner/scanning_rules/_index.md

```diff
@@ -9,6 +9,10 @@ further_reading:
   text: "Best practices for creating custom scanning rules"
 ---
 
+{{< callout url="https://www.datadoghq.com/product-preview/human-name-pii-detection-in-logs-using-machine-learning/" btn_hidden="false" >}}
+Human Name PII Detection in Logs using Machine Learning is in Preview. To enroll, click <b>Request Access</b>.
```

Comment:
```suggestion
Human Name PII Detection in Logs using machine learning is in Preview. To enroll, click <b>Request Access</b>.
```

Reply:
Thanks @rtrieu, but the full name of the feature in Preview is "Human Name PII Detection in Logs using Machine Learning", which is why I left the capitalization as it is: https://www.datadoghq.com/product-preview/human-name-pii-detection-in-logs-using-machine-learning/. What do you think?

Reply:
Otherwise, I think I'd go with sentence cap for the whole thing:
> Human name PII detection in logs using machine learning is in Preview.

Reply:
Hi May, sorry for not leaving an explanation of my original feedback, but I was thinking ML doesn't need to be capitalized since it is a mechanism. I think going sentence cap might be more in line with our style guide. I would just suggest capitalizing Logs since it is the feature/product name. But of course feel free to ignore!

> Human name detection in Logs using machine learning is in Preview.

Reply:
Thanks! I'll go all sentence cap. Also "logs" here is referencing just logs (like application logs) and not the Log Management product, so I'll keep it lowercase.

Reply:
oh d'oh, ok!